config/build_image-v0.5.2.sh

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env bash
```

Comment:
Note: This is an almost-100% rename of build.sh. The only thing different is the comment below.